cms/static/js/certificates/models/certificate.js

```diff
@@ -21,7 +21,10 @@ function (_, str, Backbone, BackboneRelational, BackboneAssociations, gettext, C
             description: 'Description of the certificate',
             course_title: 'Title of the course',
             org_logo_path: '',
-            version: 1
+            version: 1,
+
+            //only one certificate is allowed to create, default set to true.
+            singleEntity: true
         },
```

Comment:
I need to add this new field in certificate model. By using this field we can restrict user to create more then one certificate.